src/Ng/Jobs/LightningJob.cs

```diff
@@ -401,7 +442,7 @@ private void InitStrike(IDictionary<string, string> arguments)
                             if (graph.Nodes.Count() > 3000 && sortedGraphs.Count >= 20)
                             {
                                 // Process graphs
-                                await ProcessGraphsAsync(packageId, sortedGraphs);
+                                await ProcessGraphsAsync(packageId, sortedGraphs, _customDomain);
```

Comment:
You shouldn't need to pass `_customDomain` around, given that it's a field on the object.